### PR TITLE
Fix mistake about single file contracts in Spring Cloud Contracts

### PR DESCRIPTION
**Pull request by mraible**
Update [Better Integration Testing With Spring Cloud Contract](https://developer.okta.com/blog/2022/02/01/spring-cloud-contract) to fix a mistake.

### CODE REVIEW

**Review by moksamedia — APPROVED** [review]
Looks good to me.

**Review by moksamedia — APPROVED** [review]

File: _source/_posts/2022-02-01-spring-cloud-contract.md [hunk 1]
```diff
@@ -174,7 +176,7 @@ HTTP/1.1 200
 
 Next, you need to write the contracts that define the interservice agreement to govern the behavior of the producer. This is where Spring Cloud Contract comes in and things get interesting.
 
-You're going to create three contracts under `src/test/resources/contracts`. The contracts can be written in YAML or as a Groovy, Java, or Kotlin DSL script. In this tutorial I've chosen to go with the [Groovy](https://groovy-lang.org/). According to the spec, you can put multiple contracts in one file, which is what I initially did. This worked fine on the producer side, but when I tried to import the stubs on the consumer side for integration testing, it didn't work. For this reason, the three contracts below are in three separate files. 
+You're going to create three contracts under `src/test/resources/contracts`. The contracts can be written in YAML or as a Groovy, Java, or Kotlin DSL script. In this tutorial I've chosen to go with the [Groovy](https://groovy-lang.org/). These contracts could also be in a single file instead of separate files. See [the docs](https://cloud.spring.io/spring-cloud-contract/2.1.x/multi/multi_contract-dsl.html#_multiple_contracts_in_one_file).
```
Comment (marcingrzejszczak) [inline _source/_posts/2022-02-01-spring-cloud-contract.md:179]: This is a link to the old docs. You should be pointing here https://docs.spring.io/spring-cloud-contract/docs/current/reference/html/project-features.html#contract-dsl-multiple

Comment (mraible) [reply _source/_posts/2022-02-01-spring-cloud-contract.md:179]: Thanks for this @marcingrzejszczak! I created a new PR for this update. https://github.com/oktadev/okta-blog/pull/1065

Comment (moksamedia) [reply _source/_posts/2022-02-01-spring-cloud-contract.md:179]: @marcingrzejszczak Thanks!

**Pull request merged**